src/rpm.rs

```diff
@@ -27,6 +26,16 @@ pub use crate::errors::*;
 mod constants;
 pub use crate::constants::*;
 
+#[cfg(feature = "signing")]
+mod signature;
+#[cfg(feature = "signing")]
+mod signature_builder;
+
+#[cfg(feature = "signing")]
+pub use crate::signature::*;
+#[cfg(feature = "signing")]
+pub use crate::signature_builder::*;
+
 pub struct RPMPackage {
     pub metadata: RPMPackageMetadata,
     pub content: Vec<u8>,
```

Comment:
Could you define what `content` actually means? Is that the full rpm including static pre-amble, headers and the gzipped payload?

Reply:
it is just the payload, gzipped or not depends on how the rpm was build